src/NuGet.Core/NuGet.Packaging.Core/PackageType.cs

```diff
@@ -2,11 +2,16 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
+using NuGet.Shared;
 
 namespace NuGet.Packaging.Core
 {
-    public class PackageType
+    public class PackageType : IEquatable<PackageType>
     {
+        public static readonly PackageType Legacy = new PackageType("Legacy", version: new Version(0, 0));
```

Comment:
nit: you can reuse the same empty version object, or reference the one from NuGetFrameworks